poetry/utils/env.py

```diff
@@ -834,13 +834,16 @@ def remove_venv(cls, path):  # type: (Union[Path,str]) -> None
             return
         except OSError as e:
             # Continue only if e.errno == 16
-            if e.errno != 16:  # ERRNO 16: Device or resource busy
+            if (
```

Comment:
I'd like to see the `errno` module's named constants used here. A construct like `e.errno in (errno.ENOTDIR, errno.EBUSY)` would be more obvious and maintainable.